templates/pimcore5/files/.platform/services.yaml

```diff
@@ -0,0 +1,18 @@
+# The services of the project.
+#
+# Each service listed will be deployed
+# to power your Platform.sh project.
+pimcore_mysqldb:
```

Comment:
Just name it `mysqldb`.  Making it custom just makes code samples harder. :smile: